install_files/ansible-base/staging-specific.yml

```diff
@@ -6,11 +6,10 @@ dns_server: "8.8.8.8"
 ### Used by the common role ###
 ssh_users: "vagrant"
 
-# TODO Should use ansible to gather this info
-monitor_ip: "10.0.1.3"
-monitor_hostname: "mon-staging"
-app_hostname: "app-staging"
-app_ip: "10.0.1.2"
+monitor_ip: "{{ hostvars['mon-staging'].ansible_all_ipv4_addresses.0 }}"
```

Comment:
How the heck does `.0` work? That's a list (not a dict or an object) according to the Ansible [docs](https://docs.ansible.com/playbooks_variables.html#information-discovered-from-systems-facts), so shouldn't you use `ansible_all_ipv4_addresses[0]` instead?